packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -137,6 +143,16 @@ class DynamicSidecarServiceLabels(BaseModel):
         ),
     )
 
+    restart_policy: RestartPolicy = Field(
+        RestartPolicy.NO_RESTART,
+        alias="simcore.service.restart-policy",
+        description=(
+            "the dynamic-sidecar can restart all started container "
```

Comment:
```suggestion
            "the dynamic-sidecar can restart all running containers "
```

is it all or any? I guess all

Reply:
All of them are always restarted.